pkg/apis/autoscaling/v1alpha1/pa_lifecycle_test.go

```diff
@@ -1010,6 +1011,9 @@ func TestTypicalFlow(t *testing.T) {
 	// When the activator successfully forwards traffic to the deployment,
 	// we mark ourselves as active once more.
 	r.MarkActive()
+	if !r.IsActive() {
+		t.Error("Active was not set.")
+	}
```

Comment:
`apistest.CheckConditionSucceeded(r, PodAutoscalerConditionActive, t)` this is what does this. So just remove?

Reply:
Trying to ensure this function is covered